### DISCUSSION TITLE
inserts  disabling itself into with Cubase - SOLVED

### DISCUSSION THREAD

**Discussion by damienesse** [apohl79/audiogridder#483] [Help]
HI,

I have been installing and configuring AudioGridder, and I love it.
There is something weird though. After having put 5 inserts of Audio Gridder, (as sends for reverbs), when I create a new one, it opens correctly, but it automatically get "graded out" in the inserts panel, as if it was disabled. I can still load plugins in the small AudioGridder window that is opened, but once I am done, I can't do anything else, as the plugin acts as "disabled". I can't enable/disable it, and I can't open back  its window if I closed it, I can't put away the plugin either .... anybody is experiencing that?
this thing is happening on either track types: Audio Tracks, group Tracks, instrument tracks.....

Mac OS Mojave 10.14.16
Cubase 11.0.10
audioGridder 1.1.1

**Comment by CaBuffalo** [comment]
Can you try to load multiple instances instead?

I tested it with more than 5 before and AG works fine in S1 / Logic /
10.15.7.

On Fri, Apr 9, 2021 at 3:48 AM damienesse ***@***.***> wrote:

> HI,
>
> I have been installing and configuring AudioGridder, and I love it.
> There is something weird though. After having put 5 inserts of Audio
> Gridder, (as sends for reverbs), when I create a new one, it opens
> correctly, but it automatically get "graded out" in the inserts panel, as
> if it was disabled. I can still load plugins in the small AudioGridder
> window that is opened, but once I am done, I can't do anything else, as the
> plugin acts as "disabled". I can't enable/disable it, and I can't open back
> its window if I closed it, I can't put away the plugin either .... anybody
> is experiencing that?
> this thing is happening on either track types: Audio Tracks, group Tracks,
> instrument tracks.....
>
> Mac OS Mojave 10.14.16
> Cubase 11.0.10
> audioGridder 1.1.1
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/483>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AGZCHS6BULAUBRSXDTH3LQHANCNFSM42UXPAEA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by damienesse** [reply]
ok I found the the problem. This is pretty tricky.
I have a heavy template with lots of assignments and routings. When naming some specific outputs, I added the sign ~  at the beginning of the name so this family of outputs is more visible when scrolling through lists.
As soon as I moved away that sign, keeping my setup as it is, the problems dissappeared.....
Which leads me to.... not playing anymore with unusual signs in naming my tracks, outputs etc... ah ah

**Reply by apohl79** [reply]
Thx for this info. Still weird. Maybe you could still submit a bug report (https://audiogridder.com/bug-reports/), so I can have a look?

**Comment by damienesse** [comment]
Update of the SOLVED Solution.
 For people that would run into that, the problem was actually coming from the Constrain Delay Compensation button that was ON.
This button is sometime used to quick disable plugins with high latency compensation when willing to record midi instruments for example- I wasn't using it that much).
Cheers

**Comment by CaBuffalo** [comment]
Thanks for sharing the information with us.

On Sun, Apr 11, 2021 at 3:07 PM damienesse ***@***.***> wrote:

> Update of the SOLVED Solution.
> For people that would run into that, the problem was actually coming from
> the Constrain Delay Compensation button that was ON.
> This button is sometime used to quick disable plugins with high latency
> compensation when willing to record midi instruments for example- I wasn't
> using it that much).
> Cheers
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/483#discussioncomment-597348>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EFK3EWF3BYU5QU5Q3TIIMTFANCNFSM42UXPAEA>
> .
>


-- 
Kevin Ma
Supported by Gmail.
